source/radios/ft817radio.cpp

```diff
@@ -56,8 +57,11 @@ void FT817Radio::setCLARto(int frequencyHz) {
         qWarning() << "Invalid CLAR value, should be less than 10kHz";
         return;
     }
+
     char sign = frequencyHz > 0 ? 0x00 : 0x01; // CLAR sign
-    int truncated = frequencyHz / 10;          // CLAR supports 10 Hz accuracy
+    int truncated = 10 * std::floor(frequencyHz / 10.0 + 0.5);     // CLAR supports 10 Hz accuracy
```

Comment:
Based on [FT-817 user manual's Opcode Command Chart](http://www.anico.sk/manualy/YAESU/FT-817_EN.pdf), sending `12,34` sets CLAR to `12.34 kHz`.
So an input of `12340` should result in an output of `12,34`. This is why the frequency is divided by 10 and then split into two parts, (`/100` and `%100`).
Please correct me if I'm wrong here, but this was the underlying thinking behind this piece of code, when  it was written.